newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -163,6 +163,14 @@ export default class AuthenticatedUserProvider extends React.Component<
       await this._fetchUserProfileWithoutThrowingErrors();
       this._automaticallyUpdateUserProfile = true;
     } else {
+      console.info('No authenticated user found at startup.');
+      this.setState(({ authenticatedUser }) => ({
+        authenticatedUser: {
+          ...authenticatedUser,
+          receivedAssetPacks: [], // Initialize to empty array to indicate that the loading is done.
```

Comment:
This seems correct, but I wonder if it's bullet proof: should this be done as part of the logout too? Or is it done already? 
Should that be in this case factored into a "getUnauthenticatedUserState" or something that guarantees that the state is only to be modified in a single place - so that we avoid future issues with null instead of array (both are valid, but indicates a different state: not loaded or loaded)

Reply:
Really good remark. It's a bit hidden while logging out and not done properly either. 
I'll factor that 👍

Reply:
improved. I've made a few tests logging in / out and it seems good to me. Do you see any cases we should test in particular?